Meter.c

```diff
@@ -195,8 +195,8 @@ static void BarMeterMode_draw(Meter* this, int x, int y, int w) {
 
    // First draw in the bar[] buffer...
    int offset = 0;
-   int items = this->curItems;
-   for (int i = 0; i < items; i++) {
+   uint8_t items = this->curItems;
```

Comment:
No need to have this put as an extra variable. Only re-used in one further loop, should use `this->curItems` too.

Reply:
don't we save (theoretically) re-fetching `this->curItems` on every loop condition test this way?

Reply:
As this value can be shown by the compiler to be constant it will optimized to be read once anyway (if using build optimizations).

In contrast, having the extra variable adds additional indirection for the reader of the source code, which doesn't add any benefit over what a good compiler would do anyway.